webapp/src/dog/dogInterfaces.ts

```diff
@@ -9,6 +9,11 @@ import {
   TailTypes,
 } from "./dogEnums";
 
+export interface IBehaviour {
```

Comment:
Probably revert it since group C will change their behaviours return to list of strings